integration-tests/olminstall/helpers/patch_cluster_pull_secret.py

```
@@ -0,0 +1,151 @@
+    subprocess.run(["oc", "apply", "-f", "-"], input=q.stdout, check=True, text=True)
+
+    ls = subprocess.run(
+        ["oc", "get", "sa", "-n", "openshift-marketplace", "--no-headers", "-o", "custom-columns=:metadata.name"],
+        capture_output=True,
+        text=True,
+    )
+    if ls.returncode == 0:
+        for line in ls.stdout.splitlines():
+            name = line.strip()
+            if name:
+                subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)